.github/workflows/contracts.yml

```diff
@@ -34,6 +35,25 @@ jobs:
           FORKING_URL: ${{ secrets.MAINNET_ETH_HOSTNAME }}
         run: yarn test:system
 
+  contracts-deployment-dry-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
+          cache: "yarn"
+
+      - name: Cache yarn directory
+        uses: c-hive/gha-yarn-cache@v1
```

Comment:
I think this duplicates the
```
      - uses: actions/setup-node@v2
        with:
          node-version: "12.x"
          cache: "yarn"
```
We can get rid of it (here and in other places in the workflow).